helmfile/charts/calico-felix-metrics/templates/service.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Is the scrape_interval set to 30s by default?

Reply:
yes
```
- job_name: serviceMonitor/kube-system/calico-felix-metrics/0
  honor_timestamps: true
  scrape_interval: 30s
  scrape_timeout: 10s
  metrics_path: /metrics
  scheme: http
  follow_redirects: true
  relabel_configs:
```